build-support/run-clang-tidy.py

```diff
@@ -261,6 +261,9 @@ def main():
     files = [make_absolute(entry['file'], entry['directory'])
              for entry in database]
 
+    # NoisePage hack: don't check third party dependencies.
```

Comment:
Added this to run-clang-tidy.py because clang-tidy seems to barf if you stick a .clang-tidy in the `_deps/src/DEPENDENCY_NAME` folder.